test/onnx/test_utility_funs.py

```diff
@@ -591,6 +591,34 @@ def forward(self, x):
 
         np.testing.assert_allclose(ratio_pytorch, ratio_ort, rtol=0.01, atol=0.01)
 
+    @skipIfUnsupportedMinOpsetVersion(12)
```

Comment:
Maybe we can remove or replace this test. I think the test for float64 in onnxruntime tests in good enough and covers casting. But if you want to have another test, maybe put it in test_operators. Thanks.